addons/binding/org.openhab.binding.solarlog/README.md

```diff
@@ -0,0 +1,24 @@
+# openHAB Binding for SolarLog 
+
+The [SolarLog Family](http://www.solar-log.com/en/home.html) of monitoring devices for PV installations provide a MODBUS (TCP) and JSON-based API to access a number of internal data related to power generation and consumption. 
+
+This binding is an attempt to simplify the reading of these values to be displayed in openHAB. It was born out of the "frustration" that setting up SolarLog monitoring via MODBUS is almost as tedious as writing an entirely new (albeit simple) openHAB binding. And using the HTTP binding is not an option since the JSON interface requires a POST with a payload, which the HTTP binding so far does not support.
```

Comment:
This should be end user docu, I would suggest to remove this "motivational" paragraph.